src/Common/FitOnFhir.Common/Repositories/TableRepository.cs

```diff
@@ -38,11 +38,17 @@ public async Task<TEntity> GetById(string id, CancellationToken cancellationToke
         {
             EnsureArg.IsNotNullOrWhiteSpace(id, nameof(id));
 
-            var response = await _tableClient.GetEntityAsync<TableEntity>(PartitionKey, id, cancellationToken: cancellationToken);
+            try
+            {
+                var response = await _tableClient.GetEntityAsync<TableEntity>(PartitionKey, id, cancellationToken: cancellationToken);
 
-            if (response?.Value != null)
+                if (response?.Value != null)
+                {
+                    return (TEntity)Activator.CreateInstance(typeof(TEntity), response.Value);
+                }
+            }
+            catch (RequestFailedException e) when (e.Status == 404)
```

Comment:
Are we not doing anything in this catch block?

Reply:
That is correct. The only exception that we catch here is a 404 not found, and we then return null. the behavior before catching a not found was execution would halt or the caller needed to handle the 404.